### DISCUSSION TITLE
tRPC Communication in PM2 Cluster Mode or Load Balancing Deployment

### DISCUSSION THREAD

**Discussion by calebeaires** [trpc/trpc#4771] [Q&A] [answered]
I have reviewed the tRPC documentation, but I'm still unclear on how tRPC handles communication within a PM2 cluster mode server. In the case of WebSockets, we typically use the Redis module to ensure proper cluster coordination. Can someone clarify how tRPC handles this when deployed with PM2, especially in a load balancing setup?

**Comment by sachinraja** [comment, marked as answer]
I think it would just split the incoming requests across the servers in the cluster? Not sure what coordination would be required.

**Reply by calebeaires** [reply]
You are correct. I conducted some tests, and my initial observation was as follows: once connected to the front and back in RPC, the connection remained wired. It appears that this is not in line with the RPC approach.
